cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,101 @@
+import homePageObject from '../support/pages/home.pageObject';
+import settingsPageObject from '../support/pages/settings.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import faker from 'faker';
+
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+const settingsPage = new settingsPageObject();
+const homePage = new homePageObject();
+const signInPage = new SignInPageObject();
+
 describe('Settings page', () => {
+  let user;
+  const testData = {
+
+    username: faker.name.firstName(),
+    bio: faker.address.country(),
+    password: 'Qw12345678!',
+    email: faker.internet.email().toLowerCase()
+  };
+
   before(() => {
 
   });
 
   beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then(generateUser =>{
+      user = generateUser;
+    })
 
   });
 
   it('should provide an ability to update username', () => {
-
+    cy.login(user.email, user.username, user.password);
+    settingsPage.visit;
+    settingsPage.usernameField
+      .clear()
+      .type(testData.username);
+    settingsPage.updateBtn;
+    homePage.usernameLink
   });
 
   it('should provide an ability to update bio', () => {
+    cy.login(user.email, user.username, user.password);
+    settingsPage.visit;
```

Comment:
A lot of duplicated code. You can optimize it by reusing code. create different methods for it.
If the code repeats 2 and more times it is better to create a method and reuse it. If you do it in every test at the beginning so move it to beforeEach